docs/attributes-registry/android.md

```diff
@@ -0,0 +1,15 @@
+# Android
+
+**Status**: [Experimental][DocumentStatus]
+
+**type:** `android`
+
+**Description**: The Android platform on which the Android application is running.
```

Comment:
We usually do not include the above lines in the registry markdown files.

See for example https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/http.md for comparison.